app/main.py

```diff
@@ -164,5 +164,21 @@ def datasets_by_project_id(project_id):
     else:
         abort(404, description="Resource not found")
 
-
-
+@app.route("/get_email/<int:owner_id>", methods=["GET"])
+def get_owner_email(owner_id):
+    bf = Blackfynn(
+        api_token=Config.BLACKFYNN_API_TOKEN,
+        api_secret=Config.BLACKFYNN_API_SECRET,
+        env_override=False,
+        host=Config.BLACKFYNN_API_HOST
+    )
```

Comment:
It probably makes sense to instantiate the `Blackfynn` client in the global scope alongside the `s3` client. Creating the client is relatively expensive: it logs in to the platform and makes a few requests to load some initial data. If we move it to the global scope we only pay that cost once, instead of per-request.

Reply:
Good point